src/three-components/ARRenderer.js

```diff
@@ -246,28 +239,52 @@ height: 100%;`);
     return this[$outputContext];
   }
 
+  /**
+   * Calls the underlying WebXR `requestHitTest` and supports
+   * both the two- and three-argument forms, as well as
+   * implementation bugs.
+   */
+  async castRay(ray) {
```

Comment:
This handling will also need to occur in `Reticle.js` -- this should probably be moved to a util

Reply:
https://github.com/GoogleWebComponents/model-viewer/issues/281